doc/MANUAL.md

```diff
@@ -637,6 +637,10 @@ haxx,curl,7.34.0,CVE-2014-0139,MEDIUM,Unexplored,
 
 4. `--format html` - creates a report in html format according to the specified HTML theme.
 
+You can also filter scanned products by remark:
+
+![image](https://i.imgur.com/MkV7tY4.png)
```

Comment:
Oh, also, let's get this image into this commit so we don't have to rely on an external service.  With the other images we've just added them right into `doc/` but if you want to make a subdirectory just for images that might be nice.

Reply:
done!